apps/core/components/ItineraryCard/FlightTimes.js

```diff
@@ -3,6 +3,7 @@
 import * as React from 'react';
 import { View, Platform } from 'react-native';
 import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { margaritaTokens } from '@kiwicom/margarita-design-tokens';
```

Comment:
btw what about rename `margaritaTokens` export to `defaultTokens` in order to keep the same naming as in Orbit? In ideal case we could just change the import and not every occurence :)

Reply:
Then it will battle orbit tokens import and we will need to use something like
```import { defaultTokens as orbitTokens } from '@kiwicom/orbit-design-tokens'```
which will just reverse the situation.

I've been also thinking about spreading `orbit-design-tokens` directly into `margarita-design-tokens` and using just one import. 
But at the end I like it separate for now. I'm slightly worried that if we merge everything together now it will become wild west 🙂 where we can unintentionally overwrite some values from original tokens which we actually need, thanks to the similar naming.
And it can make using token values slightly more difficult. Because for example before using some color, you'll check `orbit-tokens` and even if you find it there, you'll also need to check new tokens if somebody didn't overwritten it.
But with approach in this PR, you'll still just check orbit tokens and if it's not there you'll add it to margarita tokens (if you don't already find it there).

Reply:
You are right! I didn't realise that